### PR TITLE
Support features with non-point geometry

### PR DESCRIPTION
**Pull request by brawer**

### CODE REVIEW

**Comment by brawer** [comment]
Confused about the GitHub action failure. Is it related to this change?

File: locations/exporters.py [hunk 1]
```diff
@@ -138,7 +138,8 @@ def _get_serialized_fields(self, item, default_value=None, include_empty=None):
                 )
             except ValueError:
                 logging.warning("Couldn't convert lat (%s) and lon (%s) to string", lat, lon)
-
+        elif geometry := item.get("geometry"):
+            feature.append(("geometry", geometry))
```
Comment (iandees) [inline locations/exporters.py:142]: Can you change the lat/lon stuff above to set the `geometry` property instead of adding to the feature directly?

**Comment by iandees** [comment]
> Confused about the GitHub action failure. Is it related to this change?

No, it's unrelated to your change. It seems like the Github API sometimes responds with something unexpected when trying to list the files changed in a pull request. I haven't figured out why, though.

File: locations/exporters.py (hunk 1 above)
Comment (brawer) [reply locations/exporters.py:138]: Done.

**Pull request merged**